include/ttmlir/Dialect/TTNN/IR/TTNNOpModelInterface.td

```diff
@@ -19,32 +19,25 @@ def TTNN_OpModelInterface : OpInterface<"OpModel"> {
             }],
             /*retTy=*/"size_t",
             /*methodName=*/"getOpPerfCycles",
-            /*args=*/(ins "const std::vector<TTNNLayoutAttr>&":$input_layouts, "const TTNNLayoutAttr&":$output_layout),
+            /*args=*/(ins "const std::vector<TTNNLayoutAttr>&":$inputs, "const TTNNLayoutAttr&":$output),
```

Comment:
There are some ops that might return multiple outputs. I know that maxpool2d can return the output tensor + indices of each chosen pixel in sliding window, needed for backwards pass.

Reply:
good call. Added todo, note: @odjuricicTT